src/FabCatalog.js

```diff
@@ -10,21 +11,28 @@ class Fab extends Component {
   }
 
   render() {
-    const classes = `mdc-fab
-      ${this.props.mini ? 'mdc-fab--mini ' : ''}
-      ${this.props.extended ? 'mdc-fab--extended ' : ''}
-      ${this.props.classes ? this.props.classes : ''}`;
-    const ariaLabel = this.props.ariaLabel || this.props.textLabel;
+    const {
+      className,
+      children,
+      ariaLabel,
+      textLabel,
+      mini,
+      extended,
+    } = this.props;
+    const classes = classnames('mdc-fab', className, {
```

Comment:
it looks like it was this.props.classes, what happened to that? I don't think you even need this.props.className since its not being used below.

Reply:
Renamed `classes` to `className`. That's correct we're not using this anywhere currently, but we definitely need it in future when we've more FAB variants or themeable .